core/errors/infer.cc

```diff
@@ -23,6 +23,12 @@ ErrorClass errorClassForUntyped(const GlobalState &gs, FileRef file) {
         prodHistogramInc("untyped.usages", file.id());
     }
 
+    prodCounterInc("types.input.untyped.usages");
+
+    if constexpr (sorbet::track_untyped_blame_mode) {
+        histogramInc("untyped.blames", blame.rawId());
```

Comment:
Genuine questions, curious to hear your thoughts:

- Should we use `prodHistogramInc` here instead of having `enable_counters` be always on?
- Should we run this code even if `debug_mode` is on, so that we get test coverage for this feature just by running the test suite? Right now, I'm not sure whether it's possible to write automated tests for this feature.